edge-modules/api-proxy-module/rust-sdk/azure-iot-mqtt/src/iotedge_client.rs

```diff
@@ -244,14 +221,19 @@ fn make_hyper_uri(
             Ok(url)
         }
 
-        Scheme::Unix => Ok(hyper_uds::make_hyper_uri(base, path)?),
+        Scheme::Unix => {
+            let host = hex::encode(base.as_bytes());
+            let uri = format!("unix://{}:0{}", host, path);
+            let uri = uri.parse()?;
+            Ok(uri)
```

Comment:
https://docs.rs/hyperlocal/0.8.0/hyperlocal/struct.Uri.html

Reply:
oh, that's nice!